Logging started at 2016-07-12 22:04:24
File name: C:\Users\Administrator\Dropbox\lunarisle source\.\data\logs\err\1468353863879 - Administrator - [Tue Jul 12][10.04PM].txt
System.getProperty("os.name"): "Windows Server 2012 R2"

[2016-07-12 22:04:46] java.lang.NumberFormatException: For input string: ""
[2016-07-12 22:04:46] 	at java.lang.NumberFormatException.forInputString(Unknown Source)
[2016-07-12 22:04:47] 	at java.lang.Integer.parseInt(Unknown Source)
[2016-07-12 22:04:47] 	at java.lang.Integer.parseInt(Unknown Source)
[2016-07-12 22:04:48] 	at com.vencillio.core.util.FileHandler.loadMaxPlayers(FileHandler.java:52)
[2016-07-12 22:04:48] 	at com.vencillio.core.util.FileHandler.load(FileHandler.java:24)
[2016-07-12 22:04:48] 	at com.vencillio.GameDataLoader.load(GameDataLoader.java:185)
[2016-07-12 22:04:49] 	at com.vencillio.core.GameThread.startup(GameThread.java:69)
[2016-07-12 22:04:49] 	at com.vencillio.core.GameThread.init(GameThread.java:34)
[2016-07-12 22:04:49] 	at com.vencillio.Server.main(Server.java:104)
[2016-07-12 22:59:18] java.lang.ArrayIndexOutOfBoundsException: 9999
[2016-07-12 22:59:18] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-12 22:59:18] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-12 22:59:18] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-12 22:59:18] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-12 22:59:18] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-12 22:59:18] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-12 22:59:18] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-12 23:01:04] java.lang.ArrayIndexOutOfBoundsException: 9999
[2016-07-12 23:01:04] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-12 23:01:04] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-12 23:01:04] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-12 23:01:04] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-12 23:01:04] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-12 23:01:04] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-12 23:01:04] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-12 23:08:02] java.lang.ArrayIndexOutOfBoundsException: 149
[2016-07-12 23:08:02] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-12 23:08:02] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-12 23:08:02] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-12 23:08:02] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-12 23:08:02] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-12 23:08:02] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-12 23:08:02] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-12 23:08:06] java.lang.ArrayIndexOutOfBoundsException: 149
[2016-07-12 23:08:06] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-12 23:08:06] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-12 23:08:06] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-12 23:08:06] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-12 23:08:06] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-12 23:08:06] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-12 23:08:06] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-12 23:27:16] java.lang.NullPointerException
[2016-07-12 23:27:16] 	at com.vencillio.rs2.content.DropTable.displayNpc(DropTable.java:177)
[2016-07-12 23:27:16] 	at com.vencillio.rs2.entity.player.net.in.impl.ClickButtonPacket.handle(ClickButtonPacket.java:118)
[2016-07-12 23:27:16] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-12 23:27:16] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-12 23:27:16] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-12 23:27:16] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-12 23:27:16] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-12 23:27:34] java.lang.NullPointerException
[2016-07-12 23:27:34] 	at com.vencillio.rs2.content.DropTable.displayNpc(DropTable.java:177)
[2016-07-12 23:27:34] 	at com.vencillio.rs2.entity.player.net.in.impl.ClickButtonPacket.handle(ClickButtonPacket.java:118)
[2016-07-12 23:27:34] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-12 23:27:34] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-12 23:27:34] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-12 23:27:34] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-12 23:27:34] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 00:53:46] java.lang.NullPointerException
[2016-07-13 00:53:46] 	at com.vencillio.rs2.content.combat.formula.RangeFormulas.calculateRangeAttack(RangeFormulas.java:78)
[2016-07-13 00:53:46] 	at com.vencillio.rs2.content.combat.impl.Ranged.execute(Ranged.java:41)
[2016-07-13 00:53:46] 	at com.vencillio.rs2.content.combat.Combat.attack(Combat.java:92)
[2016-07-13 00:53:46] 	at com.vencillio.rs2.content.combat.Combat.process(Combat.java:215)
[2016-07-13 00:53:46] 	at com.vencillio.rs2.entity.player.Player.process(Player.java:1545)
[2016-07-13 00:53:46] 	at com.vencillio.rs2.entity.World.process(World.java:322)
[2016-07-13 00:53:46] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 00:53:46] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 00:53:46] java.lang.NullPointerException
[2016-07-13 00:53:46] 	at com.vencillio.rs2.entity.player.net.Client.send(Client.java:329)
[2016-07-13 00:53:46] 	at com.vencillio.rs2.entity.player.net.out.impl.SendLogout.execute(SendLogout.java:13)
[2016-07-13 00:53:46] 	at com.vencillio.rs2.entity.player.Player.logout(Player.java:1471)
[2016-07-13 00:53:46] 	at com.vencillio.rs2.entity.World.process(World.java:335)
[2016-07-13 00:53:46] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 00:53:46] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 01:07:08] java.lang.NullPointerException
[2016-07-13 01:07:08] 	at com.vencillio.rs2.entity.player.net.in.command.impl.AdministratorCommand.handleCommand(AdministratorCommand.java:174)
[2016-07-13 01:07:08] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 01:07:08] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 01:07:08] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 01:07:08] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 01:07:08] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 01:07:08] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 01:07:11] java.lang.NullPointerException
[2016-07-13 01:07:11] 	at com.vencillio.rs2.entity.player.net.in.command.impl.AdministratorCommand.handleCommand(AdministratorCommand.java:174)
[2016-07-13 01:07:11] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 01:07:11] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 01:07:11] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 01:07:11] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 01:07:11] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 01:07:11] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 01:07:14] java.lang.NullPointerException
[2016-07-13 01:07:14] 	at com.vencillio.rs2.entity.player.net.in.command.impl.AdministratorCommand.handleCommand(AdministratorCommand.java:174)
[2016-07-13 01:07:14] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 01:07:14] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 01:07:14] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 01:07:14] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 01:07:14] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 01:07:14] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 01:07:17] java.lang.NullPointerException
[2016-07-13 01:07:17] 	at com.vencillio.rs2.entity.player.net.in.command.impl.AdministratorCommand.handleCommand(AdministratorCommand.java:174)
[2016-07-13 01:07:17] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 01:07:17] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 01:07:17] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 01:07:17] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 01:07:17] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 01:07:17] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 01:30:18] java.lang.NullPointerException
[2016-07-13 01:30:18] 	at com.vencillio.rs2.content.combat.formula.RangeFormulas.calculateRangeAttack(RangeFormulas.java:78)
[2016-07-13 01:30:18] 	at com.vencillio.rs2.content.combat.impl.Ranged.execute(Ranged.java:41)
[2016-07-13 01:30:18] 	at com.vencillio.rs2.content.combat.Combat.attack(Combat.java:92)
[2016-07-13 01:30:18] 	at com.vencillio.rs2.content.combat.Combat.process(Combat.java:215)
[2016-07-13 01:30:18] 	at com.vencillio.rs2.entity.player.Player.process(Player.java:1545)
[2016-07-13 01:30:18] 	at com.vencillio.rs2.entity.World.process(World.java:322)
[2016-07-13 01:30:18] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 01:30:18] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 01:30:18] java.lang.NullPointerException
[2016-07-13 01:30:18] 	at com.vencillio.rs2.entity.player.net.Client.send(Client.java:329)
[2016-07-13 01:30:18] 	at com.vencillio.rs2.entity.player.net.out.impl.SendLogout.execute(SendLogout.java:13)
[2016-07-13 01:30:18] 	at com.vencillio.rs2.entity.player.Player.logout(Player.java:1471)
[2016-07-13 01:30:18] 	at com.vencillio.rs2.entity.World.process(World.java:335)
[2016-07-13 01:30:18] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 01:30:18] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 01:31:19] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-13 01:31:19] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-13 01:31:19] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 01:31:19] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 01:31:19] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 01:31:19] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 01:31:19] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 01:31:19] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 01:31:22] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-13 01:31:22] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-13 01:31:22] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 01:31:22] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 01:31:22] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 01:31:22] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 01:31:22] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 01:31:22] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 01:37:08] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-13 01:37:08] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-13 01:37:08] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 01:37:08] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 01:37:08] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 01:37:08] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 01:37:08] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 01:37:08] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 01:37:10] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-13 01:37:10] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-13 01:37:10] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 01:37:10] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 01:37:10] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 01:37:10] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 01:37:10] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 01:37:10] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 03:50:31] java.lang.NullPointerException
[2016-07-13 03:50:31] java.lang.NullPointerException
[2016-07-13 03:50:31] 	at com.vencillio.rs2.entity.player.net.Client.send(Client.java:329)
[2016-07-13 03:50:31] 	at com.vencillio.rs2.entity.player.net.out.impl.SendLogout.execute(SendLogout.java:13)
[2016-07-13 03:50:31] 	at com.vencillio.rs2.entity.player.Player.logout(Player.java:1471)
[2016-07-13 03:50:31] 	at com.vencillio.rs2.entity.World.process(World.java:335)
[2016-07-13 03:50:31] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 03:50:31] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 05:01:08] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-13 05:01:08] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-13 05:01:08] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 05:01:08] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 05:01:08] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 05:01:08] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 05:01:08] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 05:01:08] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 05:01:14] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-13 05:01:14] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-13 05:01:14] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 05:01:14] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 05:01:14] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 05:01:14] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 05:01:14] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 05:01:14] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 05:01:26] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-13 05:01:26] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-13 05:01:26] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 05:01:26] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 05:01:26] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 05:01:26] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 05:01:26] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 05:01:26] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 05:01:34] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-13 05:01:34] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-13 05:01:34] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 05:01:34] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 05:01:34] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 05:01:34] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 05:01:34] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 05:01:34] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 05:45:25] java.lang.NullPointerException
[2016-07-13 05:45:25] java.lang.NullPointerException
[2016-07-13 06:46:01] java.lang.NullPointerException
[2016-07-13 06:46:01] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-13 06:46:01] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-13 06:46:01] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:97)
[2016-07-13 06:46:01] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 06:46:01] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 06:46:01] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 06:46:01] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 06:46:01] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 06:46:04] java.lang.NullPointerException
[2016-07-13 06:46:04] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-13 06:46:04] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-13 06:46:04] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:97)
[2016-07-13 06:46:04] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 06:46:04] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 06:46:04] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 06:46:04] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 06:46:04] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 08:42:58] java.lang.NullPointerException
[2016-07-13 08:42:58] 	at com.vencillio.rs2.content.combat.formula.MagicFormulas.getEffectiveMagicAccuracy(MagicFormulas.java:38)
[2016-07-13 08:42:58] 	at com.vencillio.rs2.content.combat.formula.MagicFormulas.getMagicAttackRoll(MagicFormulas.java:53)
[2016-07-13 08:42:58] 	at com.vencillio.rs2.content.combat.impl.Magic.finish(Magic.java:74)
[2016-07-13 08:42:58] 	at com.vencillio.rs2.content.combat.impl.Magic.execute(Magic.java:67)
[2016-07-13 08:42:58] 	at com.vencillio.rs2.content.combat.Combat.attack(Combat.java:87)
[2016-07-13 08:42:58] 	at com.vencillio.rs2.content.combat.Combat.process(Combat.java:215)
[2016-07-13 08:42:58] 	at com.vencillio.rs2.entity.player.Player.process(Player.java:1545)
[2016-07-13 08:42:58] 	at com.vencillio.rs2.entity.World.process(World.java:322)
[2016-07-13 08:42:58] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 08:42:58] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 08:42:58] java.lang.NullPointerException
[2016-07-13 08:44:31] java.lang.NullPointerException
[2016-07-13 08:44:31] 	at com.vencillio.rs2.content.combat.formula.MagicFormulas.getEffectiveMagicAccuracy(MagicFormulas.java:38)
[2016-07-13 08:44:31] 	at com.vencillio.rs2.content.combat.formula.MagicFormulas.getMagicAttackRoll(MagicFormulas.java:53)
[2016-07-13 08:44:31] 	at com.vencillio.rs2.content.combat.impl.Magic.finish(Magic.java:74)
[2016-07-13 08:44:31] 	at com.vencillio.rs2.content.combat.impl.Magic.execute(Magic.java:67)
[2016-07-13 08:44:31] 	at com.vencillio.rs2.content.combat.Combat.attack(Combat.java:87)
[2016-07-13 08:44:31] 	at com.vencillio.rs2.content.combat.Combat.process(Combat.java:215)
[2016-07-13 08:44:31] 	at com.vencillio.rs2.entity.player.Player.process(Player.java:1545)
[2016-07-13 08:44:31] 	at com.vencillio.rs2.entity.World.process(World.java:322)
[2016-07-13 08:44:31] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 08:44:31] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 08:44:31] java.lang.NullPointerException
[2016-07-13 13:01:21] java.lang.ArrayIndexOutOfBoundsException: 288
[2016-07-13 13:01:21] 	at com.vencillio.rs2.content.bank.Bank.changeTabAmount(Bank.java:415)
[2016-07-13 13:01:21] 	at com.vencillio.rs2.content.bank.Bank.withdraw(Bank.java:348)
[2016-07-13 13:01:21] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.withdrawBankItem(ItemPackets.java:1110)
[2016-07-13 13:01:21] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:167)
[2016-07-13 13:01:21] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 13:01:21] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 13:01:21] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 13:01:21] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 13:01:21] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 16:42:39] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 16:42:39] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-13 16:42:39] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-13 16:42:39] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-13 16:42:39] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-13 16:42:39] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-13 16:42:39] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-13 16:42:39] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-13 16:42:39] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-13 16:42:39] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-13 16:42:39] 	at java.lang.Thread.run(Unknown Source)
[2016-07-13 16:42:39] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 16:42:39] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-13 16:42:39] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-13 16:42:39] 	... 9 more
[2016-07-13 16:42:39] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 16:42:39] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-13 16:42:39] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-13 16:42:39] 	... 10 more
[2016-07-13 16:42:39] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 16:42:39] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-13 16:42:39] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-13 16:42:39] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-13 16:42:39] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-13 16:42:39] 	... 11 more
[2016-07-13 16:43:41] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 16:43:41] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-13 16:43:41] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-13 16:43:41] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-13 16:43:41] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-13 16:43:41] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-13 16:43:41] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-13 16:43:41] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-13 16:43:41] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-13 16:43:41] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-13 16:43:41] 	at java.lang.Thread.run(Unknown Source)
[2016-07-13 16:43:41] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 16:43:41] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-13 16:43:41] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-13 16:43:41] 	... 9 more
[2016-07-13 16:43:41] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 16:43:41] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-13 16:43:41] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-13 16:43:41] 	... 10 more
[2016-07-13 16:43:41] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 16:43:41] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-13 16:43:41] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-13 16:43:41] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-13 16:43:41] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-13 16:43:41] 	... 11 more
[2016-07-13 17:31:19] java.lang.NullPointerException
[2016-07-13 17:31:19] java.lang.NullPointerException
[2016-07-13 17:52:50] com.mashape.unirest.http.exceptions.UnirestException: java.net.SocketTimeoutException: Read timed out
[2016-07-13 17:52:50] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-13 17:52:50] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-13 17:52:50] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-13 17:52:50] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-13 17:52:50] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-13 17:52:50] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-13 17:52:50] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-13 17:52:50] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-13 17:52:50] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-13 17:52:50] 	at java.lang.Thread.run(Unknown Source)
[2016-07-13 17:52:50] Caused by: java.net.SocketTimeoutException: Read timed out
[2016-07-13 17:52:50] 	at java.net.SocketInputStream.socketRead0(Native Method)
[2016-07-13 17:52:50] 	at java.net.SocketInputStream.socketRead(Unknown Source)
[2016-07-13 17:52:50] 	at java.net.SocketInputStream.read(Unknown Source)
[2016-07-13 17:52:50] 	at java.net.SocketInputStream.read(Unknown Source)
[2016-07-13 17:52:50] 	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:136)
[2016-07-13 17:52:50] 	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:152)
[2016-07-13 17:52:50] 	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:270)
[2016-07-13 17:52:50] 	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
[2016-07-13 17:52:50] 	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
[2016-07-13 17:52:50] 	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:260)
[2016-07-13 17:52:50] 	at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:161)
[2016-07-13 17:52:50] 	at org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:153)
[2016-07-13 17:52:50] 	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:271)
[2016-07-13 17:52:50] 	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:123)
[2016-07-13 17:52:50] 	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:254)
[2016-07-13 17:52:50] 	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
[2016-07-13 17:52:50] 	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
[2016-07-13 17:52:50] 	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
[2016-07-13 17:52:50] 	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
[2016-07-13 17:52:50] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
[2016-07-13 17:52:50] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:106)
[2016-07-13 17:52:50] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
[2016-07-13 17:52:50] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:138)
[2016-07-13 17:52:50] 	... 9 more
[2016-07-13 17:59:22] java.lang.NullPointerException
[2016-07-13 17:59:22] 	at com.vencillio.rs2.content.DropTable.displayNpc(DropTable.java:177)
[2016-07-13 17:59:22] 	at com.vencillio.rs2.entity.player.net.in.impl.ClickButtonPacket.handle(ClickButtonPacket.java:118)
[2016-07-13 17:59:22] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 17:59:22] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 17:59:22] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 17:59:22] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 17:59:22] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 17:59:25] java.lang.NullPointerException
[2016-07-13 17:59:25] 	at com.vencillio.rs2.content.DropTable.displayNpc(DropTable.java:177)
[2016-07-13 17:59:25] 	at com.vencillio.rs2.entity.player.net.in.impl.ClickButtonPacket.handle(ClickButtonPacket.java:118)
[2016-07-13 17:59:25] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 17:59:25] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 17:59:25] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 17:59:25] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 17:59:25] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 18:06:58] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:06:58] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-13 18:06:58] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-13 18:06:58] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-13 18:06:58] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-13 18:06:58] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-13 18:06:58] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-13 18:06:58] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-13 18:06:58] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-13 18:06:58] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-13 18:06:58] 	at java.lang.Thread.run(Unknown Source)
[2016-07-13 18:06:58] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:06:58] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-13 18:06:58] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-13 18:06:58] 	... 9 more
[2016-07-13 18:06:58] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:06:58] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-13 18:06:58] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-13 18:06:58] 	... 10 more
[2016-07-13 18:06:58] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:06:58] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-13 18:06:58] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-13 18:06:58] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-13 18:06:58] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-13 18:06:58] 	... 11 more
[2016-07-13 18:07:46] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:07:46] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-13 18:07:46] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-13 18:07:46] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-13 18:07:46] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-13 18:07:46] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-13 18:07:46] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-13 18:07:46] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-13 18:07:46] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-13 18:07:46] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-13 18:07:46] 	at java.lang.Thread.run(Unknown Source)
[2016-07-13 18:07:46] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:07:46] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-13 18:07:46] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-13 18:07:46] 	... 9 more
[2016-07-13 18:07:46] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:07:46] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-13 18:07:46] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-13 18:07:46] 	... 10 more
[2016-07-13 18:07:46] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:07:46] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-13 18:07:46] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-13 18:07:46] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-13 18:07:46] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-13 18:07:46] 	... 11 more
[2016-07-13 18:08:16] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:08:16] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-13 18:08:16] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-13 18:08:16] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-13 18:08:16] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-13 18:08:16] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-13 18:08:16] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-13 18:08:16] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-13 18:08:16] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-13 18:08:16] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-13 18:08:16] 	at java.lang.Thread.run(Unknown Source)
[2016-07-13 18:08:16] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:08:16] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-13 18:08:16] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-13 18:08:16] 	... 9 more
[2016-07-13 18:08:16] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:08:16] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-13 18:08:16] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-13 18:08:16] 	... 10 more
[2016-07-13 18:08:16] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:08:16] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-13 18:08:16] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-13 18:08:16] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-13 18:08:16] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-13 18:08:16] 	... 11 more
[2016-07-13 18:11:09] java.lang.NullPointerException
[2016-07-13 18:11:09] 	at com.vencillio.rs2.entity.player.net.in.command.impl.AdministratorCommand.handleCommand(AdministratorCommand.java:174)
[2016-07-13 18:11:09] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 18:11:09] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 18:11:09] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 18:11:09] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 18:11:09] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 18:11:09] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 18:11:22] java.lang.NullPointerException
[2016-07-13 18:11:22] 	at com.vencillio.rs2.entity.player.net.in.command.impl.AdministratorCommand.handleCommand(AdministratorCommand.java:174)
[2016-07-13 18:11:22] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 18:11:22] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 18:11:22] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 18:11:22] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 18:11:22] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 18:11:22] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 18:11:27] java.lang.NullPointerException
[2016-07-13 18:11:27] 	at com.vencillio.rs2.entity.player.net.in.command.impl.AdministratorCommand.handleCommand(AdministratorCommand.java:174)
[2016-07-13 18:11:27] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 18:11:27] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 18:11:27] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 18:11:27] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 18:11:27] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 18:11:27] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 18:11:58] java.lang.NullPointerException
[2016-07-13 18:11:58] 	at com.vencillio.rs2.entity.player.net.in.command.impl.AdministratorCommand.handleCommand(AdministratorCommand.java:174)
[2016-07-13 18:11:58] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 18:11:58] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 18:11:58] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 18:11:58] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 18:11:58] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 18:11:58] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 18:26:31] java.lang.NullPointerException
[2016-07-13 18:26:31] 	at com.vencillio.rs2.entity.player.net.in.command.impl.AdministratorCommand.handleCommand(AdministratorCommand.java:174)
[2016-07-13 18:26:31] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 18:26:31] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 18:26:31] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 18:26:31] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 18:26:31] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 18:26:31] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 18:46:57] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:46:57] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-13 18:46:57] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-13 18:46:57] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-13 18:46:57] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-13 18:46:57] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-13 18:46:57] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-13 18:46:57] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-13 18:46:57] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-13 18:46:57] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-13 18:46:57] 	at java.lang.Thread.run(Unknown Source)
[2016-07-13 18:46:57] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:46:57] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-13 18:46:57] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-13 18:46:57] 	... 9 more
[2016-07-13 18:46:57] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:46:57] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-13 18:46:57] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-13 18:46:57] 	... 10 more
[2016-07-13 18:46:57] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:46:57] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-13 18:46:57] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-13 18:46:57] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-13 18:46:57] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-13 18:46:57] 	... 11 more
[2016-07-13 18:47:37] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:47:37] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-13 18:47:37] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-13 18:47:37] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-13 18:47:37] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-13 18:47:37] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-13 18:47:37] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-13 18:47:37] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-13 18:47:37] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-13 18:47:37] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-13 18:47:37] 	at java.lang.Thread.run(Unknown Source)
[2016-07-13 18:47:37] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:47:37] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-13 18:47:37] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-13 18:47:37] 	... 9 more
[2016-07-13 18:47:37] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:47:37] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-13 18:47:37] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-13 18:47:37] 	... 10 more
[2016-07-13 18:47:37] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 18:47:37] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-13 18:47:37] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-13 18:47:37] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-13 18:47:37] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-13 18:47:37] 	... 11 more
[2016-07-13 19:17:29] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 19:17:29] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-13 19:17:29] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-13 19:17:29] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-13 19:17:29] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-13 19:17:29] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-13 19:17:29] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-13 19:17:29] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-13 19:17:29] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-13 19:17:29] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-13 19:17:29] 	at java.lang.Thread.run(Unknown Source)
[2016-07-13 19:17:29] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 19:17:29] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-13 19:17:29] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-13 19:17:29] 	... 9 more
[2016-07-13 19:17:29] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 19:17:29] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-13 19:17:29] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-13 19:17:29] 	... 10 more
[2016-07-13 19:17:29] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 19:17:29] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-13 19:17:29] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-13 19:17:29] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-13 19:17:29] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-13 19:17:29] 	... 11 more
[2016-07-13 19:19:40] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 19:19:40] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-13 19:19:40] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-13 19:19:40] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-13 19:19:40] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-13 19:19:40] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-13 19:19:40] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-13 19:19:40] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-13 19:19:40] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-13 19:19:40] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-13 19:19:40] 	at java.lang.Thread.run(Unknown Source)
[2016-07-13 19:19:40] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 19:19:40] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-13 19:19:40] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-13 19:19:40] 	... 9 more
[2016-07-13 19:19:40] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 19:19:40] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-13 19:19:40] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-13 19:19:40] 	... 10 more
[2016-07-13 19:19:40] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-13 19:19:40] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-13 19:19:40] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-13 19:19:40] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-13 19:19:40] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-13 19:19:40] 	... 11 more
[2016-07-13 19:29:09] java.lang.NumberFormatException: For input string: "4.5"
[2016-07-13 19:29:09] 	at java.lang.NumberFormatException.forInputString(Unknown Source)
[2016-07-13 19:29:09] 	at java.lang.Long.parseLong(Unknown Source)
[2016-07-13 19:29:09] 	at java.lang.Long.parseLong(Unknown Source)
[2016-07-13 19:29:09] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:208)
[2016-07-13 19:29:09] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 19:29:09] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 19:29:09] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 19:29:09] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 19:29:09] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 19:29:09] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-13 19:29:17] java.lang.NumberFormatException: For input string: "4.5000000"
[2016-07-13 19:29:17] 	at java.lang.NumberFormatException.forInputString(Unknown Source)
[2016-07-13 19:29:17] 	at java.lang.Long.parseLong(Unknown Source)
[2016-07-13 19:29:17] 	at java.lang.Long.parseLong(Unknown Source)
[2016-07-13 19:29:17] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:208)
[2016-07-13 19:29:17] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-13 19:29:17] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-13 19:29:17] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-13 19:29:17] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-13 19:29:17] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-13 19:29:17] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
